Gui/GUIWidget.cs

```diff
@@ -1492,14 +1491,17 @@ public virtual void Initialize()
 			Initialized = true;
 		}
 
-		public int GetChildIndex(GuiWidget child)
+		public int GetChildIndex(GuiWidget childToFind)
```

Comment:
Consider if this specialized implementation can be removed in favor of .Children.IndexOf(guiWidget)

Reply:
I was under the false impression that SafeList was IList<T>. In the end it looks like the feedback had an effect though